backend/climateconnect_main/settings.py

```diff
@@ -273,7 +278,7 @@
                 {
                     "address": (env("REDIS_HOST"), env("REDIS_PORT", 6379)),
                     "password": env("REDIS_PASSWORD"),
-                    "ssl": True,
+                    "ssl": False,
```

Comment:
This seems like a change you don't want to commit

Reply:
Oh yes, definitely not 😅